lib/icinga/macroprocessor.hpp

```diff
@@ -19,8 +20,21 @@ namespace icinga
 class MacroProcessor
 {
 public:
+	struct ResolverSpec
+	{
+		String Name;
+		Object::Ptr Obj;
+
+		// Whether to resolve not only $Obj.Name$, but also just $Name$
```

Comment:
Isn't `Name = "env"` for the new resolver? So `$Name.Variable$` or `$Name.Macro$` should be more appropriate here?